src/GrooveApiClient/IGrooveClient.cs

```diff
@@ -1,310 +1,326 @@
-
-        /// <summary>
-        /// Stream a media
-        /// </summary>
-        /// <param name="id">Id of the media to be streamed</param>
-        /// <param name="clientInstanceId">Client instance Id</param>
-        /// <returns>Stream response containing the url, expiration date and content type</returns>
-        Task<StreamResponse> StreamAsync(
+            PlaylistActionType operation,
+            PlaylistAction playlistAction);
+
+        /// <summary>
+        /// Stream a media
+        /// </summary>
+        /// <param name="id">Id of the media to be streamed</param>
+        /// <param name="clientInstanceId">Client instance Id</param>
+        /// <returns>Stream response containing the url, expiration date and content type</returns>
+        Task<StreamResponse> StreamAsync(
             string id,
-            string clientInstanceId);
-
-        /// <summary>
-        /// Get a 30s preview of a media
-        /// </summary>
-        /// <param name="id">Id of the media to be streamed</param>
-        /// <param name="clientInstanceId">Client instance Id</param>
-        /// <param name="country">ISO 2 letter code.</param>
-        /// <returns>Stream response containing the url, expiration date and content type</returns>
-        Task<StreamResponse> PreviewAsync(
+            string clientInstanceId);
+
+        /// <summary>
+        /// Get a 30s preview of a media
+        /// </summary>
+        /// <param name="id">Id of the media to be streamed</param>
+        /// <param name="clientInstanceId">Client instance Id</param>
+        /// <param name="country">ISO 2 letter code.</param>
+        /// <returns>Stream response containing the url, expiration date and content type</returns>
+        Task<StreamResponse> PreviewAsync(
             string id,
             string clientInstanceId,
-            string country = null);
-
-        /// <summary>
-        /// Gets the user's profile
-        /// </summary>
-        /// <param name="mediaNamespace">"music" only for now.</param>
-        /// <param name="language">ISO 2 letter code</param>
-        /// <param name="country">ISO 2 letter code</param>
-        /// <returns></returns>
-        Task<UserProfileResponse> GetUserProfileAsync(
+            string country = null);
+
+        /// <summary>
+        /// Gets the user's profile
+        /// </summary>
+        /// <param name="mediaNamespace">"music" only for now.</param>
+        /// <param name="language">ISO 2 letter code</param>
+        /// <param name="country">ISO 2 letter code</param>
+        /// <returns></returns>
+        Task<UserProfileResponse> GetUserProfileAsync(
             MediaNamespace mediaNamespace,
             string language = null,
-            string country = null);
-    }
-}
+            string country = null);
+
+
+        Task<RadioResponse> ContinueRadioAsync(
+            MediaNamespace mediaNamespace,
+            string id,
+            int? maxItems = null);

```

Comment:
I believe it would be nice to add 2 additional overloads for this method, accepting instead of a string id a RadioResponse or a Radio, and just calling the first method with respectively the RadioResponse.SessionId and Radio.Id. This way it makes things much simpler for developers